portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,7 +35,15 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    int maxComments = Integer.parseInt(request.getParameter("max-comments"));
+    int maxComments;
+    try {
+        maxComments = Integer.parseInt(request.getParameter("max-comments"));
+    }
+    catch (Exception e) {
```

Comment:
It is good practice to catch specific exceptions that you expect might happen. If you use a generic Exception you might never find out about other bugs you have. In this case its not that bad but when your code expands a lot, it will matter.

parseInt throws a NumberFormatException.